content/applications/sales/subscriptions/products.rst

```diff
@@ -11,78 +11,108 @@ In Odoo, subscription products are also called *recurring* products.
 Configure recurrence periods
 ============================
 
-To get started with subscriptions, first make sure that the *recurrence periods* are configured as
-needed.
+To get started with subscriptions, the *recurrence periods* must be properly configured, as needed.
 
 Recurrence periods are the time periods in which subscriptions renew. They designate how often the
 customer pays for (and receives) subscription products.
 
-To configure recurrence periods, go to :menuselection:`Subscriptions --> Configuration -->
+To configure recurrence periods, go to :menuselection:`Subscriptions app --> Configuration -->
 Recurrence Periods`.
 
-The *Subscriptions* app comes with some basic recurrence periods already configured: Daily, Monthly,
-Quarterly, Weekly, Yearly, 3 Years, and 5 Years. These can be edited as needed, and any number of
-new recurrence periods can be added.
+.. image:: products/recurrence-periods-page.png
+   :align: center
+   :alt: The recurrence periods page in Odoo Subscriptions application.
+
+The *Subscriptions* app comes with some basic recurrence periods already configured:
+
+- :guilabel:`Monthly`
+- :guilabel:`Quartlery`
+- :guilabel:`Weekly`
+- :guilabel:`2 Weeks`
+- :guilabel:`Yearly`
+- :guilabel:`3 Years`
+- :guilabel:`5 Years`
 
-To create a new recurrence period, click :guilabel:`New` on the recurrence periods dashboard. Then,
-type in the :guilabel:`Name` and :guilabel:`Duration` of the recurrence period, and select the
+New recurrence periods can be added and/or edited at any time.
+
+To create a new recurrence period, click :guilabel:`New` on the :guilabel:`Reccurence Periods` page.
+Doing so reveals a blank recurrence period form.
+
+.. image:: products/recurrence-period-form.png
+   :align: center
+   :alt: A reccurence period form in Odoo Subscriptions application.
+
+Then, type in the :guilabel:`Name` and :guilabel:`Duration` of the recurrence period, and select the
 :guilabel:`Unit` that defines the duration.
 
-.. example::
-   To create a recurrence period for a subscription that will renew every two weeks, set the
-   :guilabel:`Duration` to `2` and the :guilabel:`Unit` to `Weeks`.
+.. important::
+   The unit :guilabel:`Days` *cannot* be used as a recurrence period on subscriptions. The daily
+   recurrence is meant for rentals, and *cannot* be added on recurring sales orders.
```

Comment:
```suggestion
   recurrence is meant for rentals, and **cannot** be added on recurring subscription sales orders.
```
I like that you added the call out to rentals. Since you are able to select the days unit from the drop-down, I just wanted to be as specific as possible as to why they shouldn't do it, but also "hey, there is an actual reason why that option is here, even if you shouldn't use it in this case" 😄